src/Highlight.ts

```diff
@@ -92,6 +103,18 @@ export default class Highlight {
     return this;
   }
 
+  public onFocusIn(): void {
+    if (this.options.onFocusIn) {
+      this.options.onFocusIn(this);
+    }
+  }
+
+  public onFocusOut(): void {
+    if (this.options.onFocusOut) {
+      this.options.onFocusOut(this);
+    }
+  }
+
```

Comment:
Is it all about binding `this`? I wonder what is the purpose of creating public methods, since the `focusIn`/`focusOut` functions are in `options` property.